### PR TITLE
Codestyle changes for the com_contact categories default_items.php

### PR DESCRIPTION
**Pull request by zero-24**
#### Summary of Changes

simple cs changes for com_contact categories default_items.php
#### Testing Instructions
- confirm that the frontend categories view still works after applyed patch
- quick code review as there are just simple changes :smile:

### CODE REVIEW

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 63dd10e987de10bf9a54ed87e5430a9205d90021<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10095">issues.joomla.org/joomla-cms/10095</a>.</sub>

File: components/com_contact/views/categories/tmpl/default_items.php [hunk 1]
```diff
@@ -12,52 +12,51 @@
...
 						</span>
 					<?php endif; ?>
 					<?php if (count($item->getChildren()) > 0 && $this->maxLevelcat > 1) : ?>
-						<a id="category-btn-<?php echo $item->id;?>" href="#category-<?php echo $item->id;?>"
-							data-toggle="collapse" data-toggle="button" class="btn btn-mini pull-right"><span class="icon-plus"></span></a>
-					<?php endif;?>
+						<a id="category-btn-<?php echo $item->id; ?>" href="#category-<?php echo $item->id; ?>"
+							data-toggle="collapse" data-toggle="button" class="btn btn-mini pull-right">
+							<span class="icon-plus"></span>
+						</a>
```
Comment (andrepereiradasilva) [inline components/com_contact/views/categories/tmpl/default_items.php]: Please don't add line breakings, tabs or spaces before the closing "a" tag.
Some browsers (e.g. Chrome) render that as a white space inside the link.

See this PR for instance: https://github.com/joomla/joomla-cms/pull/8206

**Comment by MATsxm** [comment]
I have tested this item :white_check_mark: successfully on 63dd10e987de10bf9a54ed87e5430a9205d90021<br /><br />No problem here - please take into account @andrepereiradasilva's notice.

Thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10095">issues.joomla.org/joomla-cms/10095</a>.</sub>

**Comment by zero-24** [comment]
Thanks @andrepereiradasilva I have just reverted that change :)

**Pull request closed without merging**